src/mds/topology/topology_item.h

```diff
@@ -861,6 +861,8 @@ bool SplitPeerId(
     uint32_t *port,
     uint32_t *idx = nullptr);
 
+bool SplitPeerId(const std::string& peerId, std::string* addr);
```

Comment:
You can add this function in 'curvefs/src/mds/topology/deal_peerid.h' or use function 'SplitPeerId' in it directly.

Reply:
> You can add this function in 'curvefs/src/mds/topology/deal_peerid.h' or use function 'SplitPeerId' in it directly.

fix